webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/DeploymentService.scala

```diff
@@ -789,7 +796,7 @@ trait PromiseGeneration_BuildNodeContext {
     , globalAgentRun        : AgentRunInterval
     , globalComplianceMode  : ComplianceMode
     , globalPolicyMode      : GlobalPolicyMode
-  ): Box[Map[NodeId, InterpolationContext]] = {
+  ): Box[NodesContextResult] = {
```

Comment:
Here, the semantic was to not stop on error, and just print a log. I'm not sure why we did that without showing the error also in generation status, because with that choice, someone can't know that a node is node updated without looking at logs, which is almost certainly never done. 
In our case, the goal is to show that there is a problem to user, so I choose to now display these error in generation status, ie: the same semantic is kept (these errors don't stop generation, as it was before that change) but error are shown in status at the end in place of success.